src/tests/example.spec.js

```diff
@@ -1,20 +1,20 @@
 // @ts-check
 const { expect } = require('@playwright/test');
 const { test } = require('../fixture');
+const { getRandomProductId } = require('../utilities/getRandomProductId');
 
 test.describe('', () => {
-    test('Perform login', async ({ loginPage, inventoryPage }) => {
+    test.beforeEach('', async ({ loginPage, inventoryPage }) => {
         await loginPage.navigate();
         await loginPage.performLogin('standard_user', 'secret_sauce');
-
         await expect(inventoryPage.headerTitle).toBeVisible();
+    });
 
+    test('Perform login', async ({ inventoryPage }) => {
```

Comment:
Is it correct name for test?

Reply:
Yes, it's a test example given by Artem, i moved login as a pre condition so it just verifies that user already logged in